CMakeLists.txt

```diff
@@ -47,6 +47,11 @@ if(NOT NO_SNOOPER_APPS)
     CACHE STRING "Disable Key Snooper for following app")
 endif()
 
+if(NOT NO_PREEDIT_APPS)
+  set(NO_PREEDIT_APPS "gvim"
```

Comment:
It's expected. gvim has some problems with preedit strings, so disable it by default.

It should be able to enable via the environmental variable `FCITX_NO_PREEDIT_APPS` but it doesn't work. I'll try to figure out why later.